go_brr.py

```diff
@@ -13,60 +13,61 @@
 def main(rooms_order: str, student_list: str, out_path: str = "out",
          html_path: str = "~/html/seating", image_path: str = "images"):
 
-    students = [tuple(s) for s in csv.reader(open(student_list))][3:]
-    random.shuffle(students)
-    student_count = len(students)
-    seat_count = 0
-    # check if the total seats is enough
-    with open(rooms_order, 'r') as f:
-        rooms = f.readlines()
-    rooms = [r for r in rooms if r != '\n']
-    rooms = list([z.strip().split() for z in rooms])
-    print(f"rooms:{rooms}")
+    with open(student_list) as s_l:
+        students = [tuple(s) for s in csv.reader(s_l)][3:]
```

Comment:
Since you've read all the data from the file into `students`, the rest of the code doesn't need to be in this context manager (the `with` block) for `student_list` (same principle as lines 22-23)